inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -256,6 +261,41 @@ class V10Parser : public IParser {
             stringToType<double>(val, value);
             adapter.set(value);
         }
+        void on_adapter(const std::string& name, ngraph::ValueAccessor<void*>& adapter) override  {
+            std::vector<std::string> value;
+            std::vector<int64_t> shape;
+            std::string el_type_str;
+            std::vector<size_t> offset;
+            std::vector<size_t> size;
+            pugi::xml_node dn = node.child("data");
+
+            if (dn.empty())
+                THROW_IE_EXCEPTION << "No attrtibutes defined for Const op!";
+
+            if (getParameters<std::string>(dn, name, value)) {
+                auto data = const_cast<char*>(reinterpret_cast<const char*>(adapter.get_ptr()));
+                size_t length = value[0].length() > adapter.size() ? adapter.size() : value[0].length();
+                value[0].copy(data, length);
+            } else if (getParameters<size_t>(dn, "offset", offset) && getParameters<size_t>(dn, "size", size)) {
```

Comment:
I propose to read offset and size only if original attribute name is "value"

Reply:
Corrected.